docs/frontend/stylelint-settings.md

```diff
@@ -0,0 +1,45 @@
+# StyleLint Settings
+This document describes stylelint setting according linter [stylelint.io](https://stylelint.io/).
+
+## Editor plugins
+You can try to find stylelint plugin in your editor package manager. For more info please [visit stylelint.io user guide](https://stylelint.io/user-guide/complementary-tools#editor-plugins)
+
+## Check and fix all files
+If you want to fix all files in your command line, you can install stylelint library from npm:
+
+``` sh
+npm install stylelint --save -dev
+```
+
+After successfull instalation, you can check and fix all files using this command:
+``` sh
+stylelint src/**/*.less --fix
```

Comment:
Usually, we use phing targets in the documentation. They should be here, do you agree?

Reply:
Yes, you are right, fixed